packages/core/src/components/popover-core/tds-popover-core.scss

```diff
@@ -0,0 +1,15 @@
+:host {
+  visibility: hidden;
+  opacity: 0;
+  transition: opacity var(--tds-motion-duration-moderate-01) var(--tds-motion-ease-in-out),
+    visibility var(--tds-motion-duration-moderate-01) var(--tds-motion-ease-in-out);
+  pointer-events: none;
+}
```

Comment:
```suggestion
:host {
  visibility: hidden;
  opacity: 0;
  pointer-events: none;
  
  &.tds-popover-animation-fade {
     transition: opacity var(--tds-motion-duration-moderate-01) var(--tds-motion-ease-in-out),
    visibility var(--tds-motion-duration-moderate-01) var(--tds-motion-ease-in-out);
  }
}
```

Reply:
Thanks for the input @mJarsater , appreciate it. But I just committed some changes I have been working on.
![meme-commit](https://github.com/user-attachments/assets/34d841c2-adee-4f89-8638-3132c59f70cc)